app/src/main/java/org/oppia/android/app/parser/StringToFractionParser.kt

```diff
@@ -29,6 +29,8 @@ class StringToFractionParser {
   fun getSubmitTimeError(text: String): FractionParsingError {
     if (invalidCharsLengthRegex.find(text) != null)
       return FractionParsingError.NUMBER_TOO_LONG
+    if (text.endsWith("/"))
+      return FractionParsingError.INVALID_FORMAT
```

Comment:
Should have braces per https://developer.android.com/kotlin/style-guide#braces. Ditto for other cases in this file that are wrong (while we're here, might as well fix them).

Reply:
Done